storage/blob_test.go

```diff
@@ -973,12 +987,12 @@ func (s *StorageBlobSuite) TestSnapshotBlobWithInvalidLease(c *chk.C) {
 	c.Assert(cli.putSingleBlockBlob(cnt, blob, []byte{}), chk.IsNil)
 
 	// generate lease.
-	_, err := cli.AcquireLease(cnt, blob, 30, "")
+	leaseID, err := cli.AcquireLease(cnt, blob, 30, "")
 	c.Assert(err, chk.IsNil)
-	c.Assert(leaseID, chk.NotNil)
+	c.Assert(leaseID, chk.Not(chk.Equals), "")
 
 	extraHeaders := map[string]string{
-		leaseID: "718e3c89-da3d-4201-b616-dd794b0bd7c1",
+		headerLeaseID: "718e3c89-da3d-4201-b616-dd794b0bd7c1",
```

Comment:
We should probably make a note of what this lease is referring to. At the moment its very Magic Number like.